src/ElvSpace.js

```diff
@@ -62,57 +62,74 @@ class ElvSpace {
       await elvAccount.Init({ privateKey: account.privateKey });
 
       if (this.debug){
-        console.log("create admin group");
+        console.log("create admin groups");
       }
 
-      let adminGroup = await elvAccount.CreateAccessGroup({
+      //Create Tenant Admin Access Group
+      let tenantAdminGroup = await elvAccount.CreateAccessGroup({
         name: `${tenantName} Tenant Admins`,
       });
 
       await elvAccount.AddToAccessGroup({
-        groupAddress: adminGroup.address,
+        groupAddress: tenantAdminGroup.address,
         accountAddress: account.address,
         isManager: true,
       });
 
-      // Add KMS to tenant admins group
-      await elvAccount.AddToAccessGroup({
-        groupAddress: adminGroup.address,
-        accountAddress: this.kmsAddress,
+      await elvAccount.SetAccountTenantAdminsAddress({
+        tenantAdminsAddress: tenantAdminGroup.address,
       });
+      account.tenantAdminsId = await elvAccount.client.userProfileClient.TenantId();
 
-      await elvAccount.SetAccountTenantAdminsAddress({
-        tenantAdminsAddress: adminGroup.address,
+      //Create Content Admin Access Group
+      let contentAdminGroup = await elvAccount.CreateAccessGroup({
+        name: `${tenantName} Content Admins`,
       });
 
-      account.tenantAdminsId = await elvAccount.client.userProfileClient.TenantId();
+      await elvAccount.AddToAccessGroup({
+        groupAddress: contentAdminGroup.address,
+        accountAddress: account.address,
+        isManager: true,
+      });
 
       if (this.debug){
-        console.log("tenant admins:", adminGroup);
+        console.log("tenant admins:", tenantAdminGroup);
+        console.log("content admins:", contentAdminGroup);
       }
+      let adminGroups = {tenantAdminGroup, contentAdminGroup};
 
       let tenant = await this.TenantDeploy({
         tenantName,
         ownerAddress: account.address,
-        adminGroupAddress: adminGroup.address,
+        tenantAdminGroupAddress: tenantAdminGroup.address,
+        contentAdminGroupAddress: contentAdminGroup.address,
       });
+
+      elvAccount.SetAccountTenantContractAddress(tenant.address);
+
```

Comment:
Here we should also add the _ELV_TENANT_ID contract metadata to the tenant admins and content admins groups.